ssh-wrapper

```diff
@@ -26,10 +26,20 @@ import sys
 
 
 def ssh(args):
-    assert args[1] == '/bin/sh'
     # WARNING: this doesn't support interactive communication, just sending
     # script through stdin in one go and return its output
-    stdin_data = sys.stdin.buffer.read()
+    start_string = "/bin/sh  << 'EOF'\n"
+    end_string = '\nEOF'
+    if args[1] == '/bin/sh':
+        stdin_data = sys.stdin.buffer.read()
+    elif args[1].startswith(start_string) and args[1].endswith(end_string):
+        stdin_data = args[1].encode('ascii', 'strict')
+        if stdin_data.find(b'\nEOF\n') != -1:
+            raise ValueError('input is not single heredoc')
+        stdin_data = stdin_data[len(start_string):1 - len(end_string)]
```

Comment:
I'd prefer moving this above the second end marker check, which will simplify the check (re-use end_string directly). I'll adjust in a separate commit.

Reply:
Hmm on the other hand, it may be better to simply check for `end_string + '\n'`, but keep the check where it is.